src/web/waiters/RecipeWaiter.mjs

```diff
@@ -33,17 +33,19 @@ class RecipeWaiter {
      */
     initialiseOperationDragNDrop() {
         const recList = document.getElementById("rec-list");
+        const isMobileView = window.innerWidth < this.app.breakpoint;
```

Comment:
I've seen the expression above being used five times in total. Might be valuable to write a single method for it that's used throughout the application.
```js
isMobileView() {
  return window.innerWidth < this.app.breakpoint;
}
```